pkg/daemon/ceph/client/info.go

```diff
@@ -36,13 +36,14 @@ import (
 // ClusterInfo is a collection of information about a particular Ceph cluster. Rook uses information
 // about the cluster to configure daemons to connect to the desired cluster.
 type ClusterInfo struct {
-	FSID          string
-	MonitorSecret string
-	CephCred      CephCred
-	Monitors      map[string]*MonInfo
-	CephVersion   cephver.CephVersion
-	Namespace     string
-	OwnerInfo     *k8sutil.OwnerInfo
+	FSID             string
+	MonitorSecret    string
+	CephCred         CephCred
+	InternalMonitors map[string]*MonInfo
+	ExtArbiterMons   map[string]*MonInfo
```

Comment:
Let's reserve the term "arbiter" only for stretch clusters. Since this feature is not for stretch clusters, let's just call them external mons.
```suggestion
	ExternalMons   map[string]*MonInfo
```

Reply:
sure, will remove arbiter naming from all places